intlogger.go

```diff
@@ -635,8 +635,21 @@ func (l *intLogger) StandardLogger(opts *StandardLoggerOptions) *log.Logger {
 }
 
 func (l *intLogger) StandardWriter(opts *StandardLoggerOptions) io.Writer {
+        newLog := &intLogger{
+		json:              l.json,
+		name:              l.name,
+		timeFormat:        l.timeFormat,
+		mutex:             l.mutex,
+		writer:            l.writer,
+		level:             l.level,
+		exclude:           l.exclude,
+		independentLevels: l.independentLevels,
+	}
```

Comment:
```suggestion
        newLog := *l
```

Reply:
We both use hclog and standlog in one project.
Because the callerOffset must plus 4, so we have to use another intLogger, but not a reference to the old one.
If we only use hclog or standlog, then the code can be simplificated.

Reply:
Can you please also pull this change in?

Reply:
If pull this change in, then some one may not use hclog and standlog together.
If you sure about it, I will pull it.